### DISCUSSION TITLE
ARGOCD UI is not showing apps in correct manner

### DISCUSSION THREAD

**Discussion by AnubhavSabarwal** [argoproj/argo-cd#17327] [General]
After upgrading to 2.10.0, the ARGOCD UI is not showing the apps in correct manner.

![Uploading Screenshot 2024-02-27 at 10.01.09 PM.png…]()

**Comment by FernandoMiguel** [comment]
@AnubhavSabarwal no screenshot

**Comment by AnubhavSabharwa** [comment]
This has been fixed in 2.10.2, it was known issue
